### PR TITLE
EOS-14514 : Include serial number in manifest support bundle

### PR DESCRIPTION
**Pull request by satish-seagate**
## Problem Statement
<pre>
  <code>
    Improvment :  Restructured summery manifest file and added lr-serial-number.
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Improvment :  Restructured summery manifest file and added lr-serial-number.
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
  </code>
</pre>

### CODE REVIEW

File: low-level/files/opt/seagate/sspl/bin/manifest_support_bundle [hunk 1]
```diff
@@ -23,213 +23,230 @@ import sys
 import socket
 import json
 import subprocess
+from sspl_constants import PRODUCT_FAMILY, node_key_id
 
-from sspl_constants import PRODUCT_FAMILY
-
-localTempPath = "/tmp/support_bundle/"
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/bin/manifest_support_bundle:29]: this whole code restructuring is making big change set visible, very hard to track what all changed. I believe putting these vars & functions inside class & then adding proper spaces are only involved.

File: low-level/files/opt/seagate/sspl/bin/manifest_support_bundle [hunk 2]
```diff
@@ -23,213 +23,230 @@ import sys
...
+        if not os.path.exists(self.localTempPath):
+            os.makedirs(self.localTempPath)
         try:
-            sspl_state=open(f"/var/{PRODUCT_FAMILY}/sspl/data/state.txt").readline().rstrip().split("=")
-            # Capturing enclosure data only when sspl state is active
-            sspl_state = sspl_state[1] if len(sspl_state)==2 else ''
-            if sspl_state == 'active':
+            sspl_state = ""
+            try:
+                lr_serial_number = open(self.lr_serial_number_path).readline().rstrip()
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/bin/manifest_support_bundle:66]: this is the only change which we seeked through this patch majorly

**Review by cdeshmukh — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/bin/manifest_support_bundle (hunk 1 above)
Comment (satish-seagate) [reply low-level/files/opt/seagate/sspl/bin/manifest_support_bundle:29]: Yes @cdeshmukh

File: low-level/files/opt/seagate/sspl/bin/manifest_support_bundle (hunk 2 above)
Comment (satish-seagate) [reply low-level/files/opt/seagate/sspl/bin/manifest_support_bundle:66]: Yes

**Review by saurabhkhanvilkar — APPROVED** [review]

**Review by indrajitzagade — APPROVED** [review]
Approving based on other reviews.

**Pull request merged**